eth-custodian/contracts/ProofKeeper.sol

```diff
@@ -9,16 +9,16 @@ contract ProofKeeper {
     using ProofDecoder for Borsh.Data;
 
     INearProver public prover_;
-    bytes public nearProofProducer_;
+    bytes public nearProofProducerAccount_;
```

Comment:
I'm still getting used to solidity style guidelines. They suggest [using ending underscores](https://docs.soliditylang.org/en/v0.8.1/style-guide.html#avoiding-naming-collisions) to avoid name collision, however we should use it consistently. Previously you used `admin` for state variable and `admin_` for constructor input, lets use that convention.

This also needs to be changed in token-connector eventually, since it is not enforces by linter.

Reply:
I agree that it's better to use it without underscores. The underscores policy migrated to `ProofKeeper` form the old `Locker` to make it consistent with the token-connector. So I like that we will get rid of it.
Btw I actually used `_admin` for the input parameter as I think underscore at the beginning is better for avoiding collisions.